ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/QueryDescription.java

```diff
@@ -110,6 +121,10 @@ public String getExecutionPlan() {
     return state;
   }
 
+  public Map<KsqlHostInfoEntity, String> getKsqlHostQueryState() {
+    return ksqlHostQueryState;
```

Comment:
Exposes mutable state of object, i.e. breaks encapsulation. I think this should still ensure consistent ordering:

```suggestion
    return Collections.unmodifyableMap(ksqlHostQueryState);
```

If not, then you can always just store a normal map in the field and return:

```java
  final Map<x, y> orderedMap = new TreeMap<>(Comparator.comparing(KsqlHostInfoEntity::toString))
   orderedMap.putAll(ksqlHostQueryState);
return orderedMap;
```